cmd/sonobuoy/app/operations/status.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operations
+
+import (
+// TODO
+)
+
+// Status determines the status of the sonobuoy run in order to assist the user.
+func GetStatus( /*opts?*/ ) error {
```

Comment:
maybe an empty struct would make sense here, to make everything more uniform?

Reply:
I'll let you decide later ;-)